src/main/java/uk/ac/ebi/pride/utilities/data/filter/NoPeptideFilter.java

```diff
@@ -0,0 +1,20 @@
+package uk.ac.ebi.pride.utilities.data.filter;
+
+import uk.ac.ebi.pride.utilities.data.core.Peptide;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author ntoro
+ * @since 25/02/15 17:23
+ */
+public class NoPeptideFilter implements PeptideFilter {
```

Comment:
@noedelta check if we can skip memory and another step of moving peptides from one array to another one.